packages/@okta/vuepress-site/docs/reference/api/users/index.md

```diff
@@ -1624,7 +1624,7 @@ in the request is deleted.
 
 `profile` and `credentials` can be updated independently or together with a single request.
 
->**Note:** Currently, the User Type of a user cannot be changed. If the Request Parameters include the `type` element from the [User object](#user-object), the value must match the existing type of the user. To change a User's type, the User object must be deleted and recreated with the desired Type.
+>**Note:** Currently, the User Type of a user can only be changed via a full replace. If the Request Parameters include the `type` element from the [User object](#user-object), on a partial update the value must match the existing type of the user. Only administrators are permitted to change the user type of a user; end users are not allowed to change their own user type. 
```

Comment:
Do we still need this note? I feel like it would be better to just specify under the partial update portion that type cannot be changed, and specify under the full update portion that it can be change.d

Reply:
Hm, I feel there's some value to the bit about how it's okay to include a type in a partial update but only if the type is not changing.

Reply:
This is the only place in the whole API, as far as I know, where changing a thing (the user type) can only be done via a full update, and the partial update equivalent of the same API does not support it.  Calling that out explicitly seems prudent.  It is a fairly odd and unexpected thing to do.

Reply:
I agree the full/partial distinction is worth keeping.  Nit: I think the second sentence flows a bit more smoothly, and makes the contrast with "full replace" more obvious, if the "partial update" part is moved earlier:

... via a full replace.  If the Request Parameters on a partial update include the `type` element from the ..., the value must ...